{"error":{},"level":"error","message":"uncaughtException: app.use() requires a middleware function\nTypeError: app.use() requires a middleware function\n    at Function.use (/Users/adavia/Projects/node/statement/backend/node_modules/express/lib/application.js:210:11)\n    at Object.<anonymous> (/Users/adavia/Projects/node/statement/backend/src/index.js:16:5)\n    at Module._compile (module.js:660:30)\n    at loader (/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js:144:5)\n    at Object.require.extensions.(anonymous function) [as .js] (/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js:154:7)\n    at Module.load (module.js:573:32)\n    at tryModuleLoad (module.js:513:12)\n    at Function.Module._load (module.js:505:3)\n    at Function.Module.runMain (module.js:701:10)\n    at Object.<anonymous> (/Users/adavia/Projects/node/statement/backend/node_modules/babel-cli/lib/_babel-node.js:154:22)\n    at Module._compile (module.js:660:30)\n    at Object.Module._extensions..js (module.js:671:10)\n    at Module.load (module.js:573:32)\n    at tryModuleLoad (module.js:513:12)\n    at Function.Module._load (module.js:505:3)\n    at Function.Module.runMain (module.js:701:10)","stack":"TypeError: app.use() requires a middleware function\n    at Function.use (/Users/adavia/Projects/node/statement/backend/node_modules/express/lib/application.js:210:11)\n    at Object.<anonymous> (/Users/adavia/Projects/node/statement/backend/src/index.js:16:5)\n    at Module._compile (module.js:660:30)\n    at loader (/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js:144:5)\n    at Object.require.extensions.(anonymous function) [as .js] (/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js:154:7)\n    at Module.load (module.js:573:32)\n    at tryModuleLoad (module.js:513:12)\n    at Function.Module._load (module.js:505:3)\n    at Function.Module.runMain (module.js:701:10)\n    at Object.<anonymous> (/Users/adavia/Projects/node/statement/backend/node_modules/babel-cli/lib/_babel-node.js:154:22)\n    at Module._compile (module.js:660:30)\n    at Object.Module._extensions..js (module.js:671:10)\n    at Module.load (module.js:573:32)\n    at tryModuleLoad (module.js:513:12)\n    at Function.Module._load (module.js:505:3)\n    at Function.Module.runMain (module.js:701:10)","exception":true,"date":"Wed Sep 19 2018 14:50:49 GMT-0500 (EST)","process":{"pid":7786,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":151736320,"heapTotal":130211840,"heapUsed":105031512,"external":37077067}},"os":{"loadavg":[2.4130859375,2.24853515625,2.33740234375],"uptime":88849},"trace":[{"column":11,"file":"/Users/adavia/Projects/node/statement/backend/node_modules/express/lib/application.js","function":"Function.use","line":210,"method":"use","native":false},{"column":5,"file":"/Users/adavia/Projects/node/statement/backend/src/index.js","function":null,"line":16,"method":null,"native":false},{"column":30,"file":"module.js","function":"Module._compile","line":660,"method":"_compile","native":false},{"column":5,"file":"/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js","function":"loader","line":144,"method":null,"native":false},{"column":7,"file":"/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js","function":"Object.require.extensions.(anonymous function) [as .js]","line":154,"method":"js]","native":false},{"column":32,"file":"module.js","function":"Module.load","line":573,"method":"load","native":false},{"column":12,"file":"module.js","function":"tryModuleLoad","line":513,"method":null,"native":false},{"column":3,"file":"module.js","function":"Module._load","line":505,"method":"_load","native":false},{"column":10,"file":"module.js","function":"Module.runMain","line":701,"method":"runMain","native":false},{"column":22,"file":"/Users/adavia/Projects/node/statement/backend/node_modules/babel-cli/lib/_babel-node.js","function":null,"line":154,"method":null,"native":false},{"column":30,"file":"module.js","function":"Module._compile","line":660,"method":"_compile","native":false},{"column":10,"file":"module.js","function":"Module._extensions..js","line":671,"method":".js","native":false},{"column":32,"file":"module.js","function":"Module.load","line":573,"method":"load","native":false},{"column":12,"file":"module.js","function":"tryModuleLoad","line":513,"method":null,"native":false},{"column":3,"file":"module.js","function":"Module._load","line":505,"method":"_load","native":false},{"column":10,"file":"module.js","function":"Module.runMain","line":701,"method":"runMain","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Wed Sep 19 2018 16:13:08 GMT-0500 (EST)","process":{"pid":8256,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":148738048,"heapTotal":117153792,"heapUsed":73662888,"external":41120795}},"os":{"loadavg":[2.10888671875,2.28759765625,2.2607421875],"uptime":93788},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Wed Sep 19 2018 16:13:08 GMT-0500 (EST)","process":{"pid":8256,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":148766720,"heapTotal":117153792,"heapUsed":73801000,"external":41120819}},"os":{"loadavg":[2.10888671875,2.28759765625,2.2607421875],"uptime":93788},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Wed Sep 19 2018 16:13:08 GMT-0500 (EST)","process":{"pid":8256,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":148819968,"heapTotal":117153792,"heapUsed":74451392,"external":41122598}},"os":{"loadavg":[2.10888671875,2.28759765625,2.2607421875],"uptime":93788},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Wed Sep 19 2018 16:13:08 GMT-0500 (EST)","process":{"pid":8256,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":148819968,"heapTotal":117153792,"heapUsed":74501992,"external":41130790}},"os":{"loadavg":[2.10888671875,2.28759765625,2.2607421875],"uptime":93788},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Wed Sep 19 2018 16:13:09 GMT-0500 (EST)","process":{"pid":8256,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":148819968,"heapTotal":117153792,"heapUsed":74564152,"external":41130790}},"os":{"loadavg":[2.10888671875,2.28759765625,2.2607421875],"uptime":93789},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Wed Sep 19 2018 16:13:09 GMT-0500 (EST)","process":{"pid":8256,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":148819968,"heapTotal":117153792,"heapUsed":74610176,"external":41130790}},"os":{"loadavg":[2.10888671875,2.28759765625,2.2607421875],"uptime":93789},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Wed Sep 19 2018 16:13:09 GMT-0500 (EST)","process":{"pid":8256,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":148824064,"heapTotal":117153792,"heapUsed":74671272,"external":41130790}},"os":{"loadavg":[2.10888671875,2.28759765625,2.2607421875],"uptime":93789},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Wed Sep 19 2018 16:13:09 GMT-0500 (EST)","process":{"pid":8256,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":148824064,"heapTotal":117153792,"heapUsed":74716768,"external":41138982}},"os":{"loadavg":[2.10888671875,2.28759765625,2.2607421875],"uptime":93789},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Wed Sep 19 2018 16:13:10 GMT-0500 (EST)","process":{"pid":8256,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":148824064,"heapTotal":117153792,"heapUsed":74777136,"external":41138982}},"os":{"loadavg":[2.10888671875,2.28759765625,2.2607421875],"uptime":93790},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Wed Sep 19 2018 16:13:10 GMT-0500 (EST)","process":{"pid":8256,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":148824064,"heapTotal":117153792,"heapUsed":74837848,"external":41138982}},"os":{"loadavg":[2.10888671875,2.28759765625,2.2607421875],"uptime":93790},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Wed Sep 19 2018 16:13:12 GMT-0500 (EST)","process":{"pid":8256,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":148824064,"heapTotal":117153792,"heapUsed":74898488,"external":41138982}},"os":{"loadavg":[2.10888671875,2.28759765625,2.2607421875],"uptime":93792},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Wed Sep 19 2018 16:13:12 GMT-0500 (EST)","process":{"pid":8256,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":148824064,"heapTotal":117153792,"heapUsed":74944424,"external":41147174}},"os":{"loadavg":[2.10888671875,2.28759765625,2.2607421875],"uptime":93792},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Wed Sep 19 2018 16:13:15 GMT-0500 (EST)","process":{"pid":8256,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":148824064,"heapTotal":117153792,"heapUsed":75022288,"external":41147434}},"os":{"loadavg":[2.10009765625,2.28271484375,2.2587890625],"uptime":93795},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Wed Sep 19 2018 16:13:15 GMT-0500 (EST)","process":{"pid":8256,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":148824064,"heapTotal":117153792,"heapUsed":75067704,"external":41147434}},"os":{"loadavg":[2.10009765625,2.28271484375,2.2587890625],"uptime":93795},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Wed Sep 19 2018 16:13:20 GMT-0500 (EST)","process":{"pid":8256,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":148828160,"heapTotal":117153792,"heapUsed":75140128,"external":41147694}},"os":{"loadavg":[2.01171875,2.26123046875,2.2509765625],"uptime":93800},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Wed Sep 19 2018 16:13:20 GMT-0500 (EST)","process":{"pid":8256,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":148828160,"heapTotal":117153792,"heapUsed":75185624,"external":41155886}},"os":{"loadavg":[2.01171875,2.26123046875,2.2509765625],"uptime":93800},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Wed Sep 19 2018 16:13:28 GMT-0500 (EST)","process":{"pid":8256,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":106487808,"heapTotal":73568256,"heapUsed":68654472,"external":35725752}},"os":{"loadavg":[2.01611328125,2.25244140625,2.24755859375],"uptime":93808},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Wed Sep 19 2018 16:13:28 GMT-0500 (EST)","process":{"pid":8256,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":106545152,"heapTotal":73568256,"heapUsed":68717800,"external":35725752}},"os":{"loadavg":[2.01611328125,2.25244140625,2.24755859375],"uptime":93808},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Wed Sep 19 2018 16:13:42 GMT-0500 (EST)","process":{"pid":8256,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":106659840,"heapTotal":73568256,"heapUsed":68836296,"external":35726532}},"os":{"loadavg":[2.75,2.40673828125,2.302734375],"uptime":93822},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Wed Sep 19 2018 16:13:42 GMT-0500 (EST)","process":{"pid":8256,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":106708992,"heapTotal":73568256,"heapUsed":68886816,"external":35726532}},"os":{"loadavg":[2.75,2.40673828125,2.302734375],"uptime":93822},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Wed Sep 19 2018 16:14:06 GMT-0500 (EST)","process":{"pid":8256,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":98283520,"heapTotal":73568256,"heapUsed":69003592,"external":35736024}},"os":{"loadavg":[2.2255859375,2.30859375,2.26953125],"uptime":93846},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Wed Sep 19 2018 16:14:06 GMT-0500 (EST)","process":{"pid":8256,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":98328576,"heapTotal":73568256,"heapUsed":69048976,"external":35736024}},"os":{"loadavg":[2.2255859375,2.30859375,2.26953125],"uptime":93846},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"code":"MODULE_NOT_FOUND"},"level":"error","message":"uncaughtException: Cannot find module '../controllers/categories.controller'\nError: Cannot find module '../controllers/categories.controller'\n    at Function.Module._resolveFilename (module.js:555:15)\n    at Function.Module._load (module.js:482:25)\n    at Module.require (module.js:604:17)\n    at require (internal/module.js:11:18)\n    at Object.<anonymous> (/Users/adavia/Projects/node/statement/backend/src/routes/categories.route.js:2:1)\n    at Module._compile (module.js:660:30)\n    at loader (/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js:144:5)\n    at Object.require.extensions.(anonymous function) [as .js] (/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js:154:7)\n    at Module.load (module.js:573:32)\n    at tryModuleLoad (module.js:513:12)\n    at Function.Module._load (module.js:505:3)\n    at Module.require (module.js:604:17)\n    at require (internal/module.js:11:18)\n    at Object.<anonymous> (/Users/adavia/Projects/node/statement/backend/src/routes/index.js:5:1)\n    at Module._compile (module.js:660:30)\n    at loader (/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js:144:5)","stack":"Error: Cannot find module '../controllers/categories.controller'\n    at Function.Module._resolveFilename (module.js:555:15)\n    at Function.Module._load (module.js:482:25)\n    at Module.require (module.js:604:17)\n    at require (internal/module.js:11:18)\n    at Object.<anonymous> (/Users/adavia/Projects/node/statement/backend/src/routes/categories.route.js:2:1)\n    at Module._compile (module.js:660:30)\n    at loader (/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js:144:5)\n    at Object.require.extensions.(anonymous function) [as .js] (/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js:154:7)\n    at Module.load (module.js:573:32)\n    at tryModuleLoad (module.js:513:12)\n    at Function.Module._load (module.js:505:3)\n    at Module.require (module.js:604:17)\n    at require (internal/module.js:11:18)\n    at Object.<anonymous> (/Users/adavia/Projects/node/statement/backend/src/routes/index.js:5:1)\n    at Module._compile (module.js:660:30)\n    at loader (/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js:144:5)","exception":true,"date":"Thu Sep 20 2018 09:22:59 GMT-0500 (EST)","process":{"pid":9048,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":151961600,"heapTotal":133881856,"heapUsed":102710416,"external":35784240}},"os":{"loadavg":[2.58544921875,10.95068359375,16.48681640625],"uptime":155554},"trace":[{"column":15,"file":"module.js","function":"Module._resolveFilename","line":555,"method":"_resolveFilename","native":false},{"column":25,"file":"module.js","function":"Module._load","line":482,"method":"_load","native":false},{"column":17,"file":"module.js","function":"Module.require","line":604,"method":"require","native":false},{"column":18,"file":"internal/module.js","function":"require","line":11,"method":null,"native":false},{"column":1,"file":"/Users/adavia/Projects/node/statement/backend/src/routes/categories.route.js","function":null,"line":2,"method":null,"native":false},{"column":30,"file":"module.js","function":"Module._compile","line":660,"method":"_compile","native":false},{"column":5,"file":"/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js","function":"loader","line":144,"method":null,"native":false},{"column":7,"file":"/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js","function":"Object.require.extensions.(anonymous function) [as .js]","line":154,"method":"js]","native":false},{"column":32,"file":"module.js","function":"Module.load","line":573,"method":"load","native":false},{"column":12,"file":"module.js","function":"tryModuleLoad","line":513,"method":null,"native":false},{"column":3,"file":"module.js","function":"Module._load","line":505,"method":"_load","native":false},{"column":17,"file":"module.js","function":"Module.require","line":604,"method":"require","native":false},{"column":18,"file":"internal/module.js","function":"require","line":11,"method":null,"native":false},{"column":1,"file":"/Users/adavia/Projects/node/statement/backend/src/routes/index.js","function":null,"line":5,"method":null,"native":false},{"column":30,"file":"module.js","function":"Module._compile","line":660,"method":"_compile","native":false},{"column":5,"file":"/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js","function":"loader","line":144,"method":null,"native":false}]}
{"error":{},"level":"error","message":"uncaughtException: Route.get() requires a callback function but got a [object Undefined]\nError: Route.get() requires a callback function but got a [object Undefined]\n    at Route.(anonymous function) [as get] (/Users/adavia/Projects/node/statement/backend/node_modules/express/lib/router/route.js:202:15)\n    at Function.proto.(anonymous function) [as get] (/Users/adavia/Projects/node/statement/backend/node_modules/express/lib/router/index.js:510:19)\n    at Object.<anonymous> (/Users/adavia/Projects/node/statement/backend/src/routes/categories.route.js:6:8)\n    at Module._compile (module.js:660:30)\n    at loader (/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js:144:5)\n    at Object.require.extensions.(anonymous function) [as .js] (/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js:154:7)\n    at Module.load (module.js:573:32)\n    at tryModuleLoad (module.js:513:12)\n    at Function.Module._load (module.js:505:3)\n    at Module.require (module.js:604:17)\n    at require (internal/module.js:11:18)\n    at Object.<anonymous> (/Users/adavia/Projects/node/statement/backend/src/routes/index.js:5:1)\n    at Module._compile (module.js:660:30)\n    at loader (/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js:144:5)\n    at Object.require.extensions.(anonymous function) [as .js] (/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js:154:7)\n    at Module.load (module.js:573:32)","stack":"Error: Route.get() requires a callback function but got a [object Undefined]\n    at Route.(anonymous function) [as get] (/Users/adavia/Projects/node/statement/backend/node_modules/express/lib/router/route.js:202:15)\n    at Function.proto.(anonymous function) [as get] (/Users/adavia/Projects/node/statement/backend/node_modules/express/lib/router/index.js:510:19)\n    at Object.<anonymous> (/Users/adavia/Projects/node/statement/backend/src/routes/categories.route.js:6:8)\n    at Module._compile (module.js:660:30)\n    at loader (/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js:144:5)\n    at Object.require.extensions.(anonymous function) [as .js] (/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js:154:7)\n    at Module.load (module.js:573:32)\n    at tryModuleLoad (module.js:513:12)\n    at Function.Module._load (module.js:505:3)\n    at Module.require (module.js:604:17)\n    at require (internal/module.js:11:18)\n    at Object.<anonymous> (/Users/adavia/Projects/node/statement/backend/src/routes/index.js:5:1)\n    at Module._compile (module.js:660:30)\n    at loader (/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js:144:5)\n    at Object.require.extensions.(anonymous function) [as .js] (/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js:154:7)\n    at Module.load (module.js:573:32)","exception":true,"date":"Thu Sep 20 2018 09:23:30 GMT-0500 (EST)","process":{"pid":9053,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":152838144,"heapTotal":133881856,"heapUsed":99203448,"external":35803270}},"os":{"loadavg":[2.5302734375,10.1279296875,15.99609375],"uptime":155585},"trace":[{"column":15,"file":"/Users/adavia/Projects/node/statement/backend/node_modules/express/lib/router/route.js","function":"Route.(anonymous function) [as get]","line":202,"method":"(anonymous function) [as get]","native":false},{"column":19,"file":"/Users/adavia/Projects/node/statement/backend/node_modules/express/lib/router/index.js","function":"Function.proto.(anonymous function) [as get]","line":510,"method":"(anonymous function) [as get]","native":false},{"column":8,"file":"/Users/adavia/Projects/node/statement/backend/src/routes/categories.route.js","function":null,"line":6,"method":null,"native":false},{"column":30,"file":"module.js","function":"Module._compile","line":660,"method":"_compile","native":false},{"column":5,"file":"/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js","function":"loader","line":144,"method":null,"native":false},{"column":7,"file":"/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js","function":"Object.require.extensions.(anonymous function) [as .js]","line":154,"method":"js]","native":false},{"column":32,"file":"module.js","function":"Module.load","line":573,"method":"load","native":false},{"column":12,"file":"module.js","function":"tryModuleLoad","line":513,"method":null,"native":false},{"column":3,"file":"module.js","function":"Module._load","line":505,"method":"_load","native":false},{"column":17,"file":"module.js","function":"Module.require","line":604,"method":"require","native":false},{"column":18,"file":"internal/module.js","function":"require","line":11,"method":null,"native":false},{"column":1,"file":"/Users/adavia/Projects/node/statement/backend/src/routes/index.js","function":null,"line":5,"method":null,"native":false},{"column":30,"file":"module.js","function":"Module._compile","line":660,"method":"_compile","native":false},{"column":5,"file":"/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js","function":"loader","line":144,"method":null,"native":false},{"column":7,"file":"/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js","function":"Object.require.extensions.(anonymous function) [as .js]","line":154,"method":"js]","native":false},{"column":32,"file":"module.js","function":"Module.load","line":573,"method":"load","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Fri Sep 21 2018 17:01:45 GMT-0500 (EST)","process":{"pid":11842,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":22196224,"heapTotal":81956864,"heapUsed":74765200,"external":35927246}},"os":{"loadavg":[2.1337890625,2.1650390625,2.33544921875],"uptime":269437},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Fri Sep 21 2018 17:01:45 GMT-0500 (EST)","process":{"pid":11842,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":23535616,"heapTotal":81956864,"heapUsed":74893376,"external":35927270}},"os":{"loadavg":[2.1337890625,2.1650390625,2.33544921875],"uptime":269437},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Fri Sep 21 2018 17:01:45 GMT-0500 (EST)","process":{"pid":11842,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":23670784,"heapTotal":81956864,"heapUsed":74975248,"external":35927530}},"os":{"loadavg":[2.1337890625,2.1650390625,2.33544921875],"uptime":269437},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Fri Sep 21 2018 17:01:45 GMT-0500 (EST)","process":{"pid":11842,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":23728128,"heapTotal":81956864,"heapUsed":75023960,"external":35935722}},"os":{"loadavg":[2.1337890625,2.1650390625,2.33544921875],"uptime":269437},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Fri Sep 21 2018 17:01:46 GMT-0500 (EST)","process":{"pid":11842,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":23818240,"heapTotal":81956864,"heapUsed":75094664,"external":35935722}},"os":{"loadavg":[2.1337890625,2.1650390625,2.33544921875],"uptime":269438},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Fri Sep 21 2018 17:01:46 GMT-0500 (EST)","process":{"pid":11842,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":23859200,"heapTotal":81956864,"heapUsed":75138712,"external":35935722}},"os":{"loadavg":[2.1337890625,2.1650390625,2.33544921875],"uptime":269438},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Fri Sep 21 2018 17:01:47 GMT-0500 (EST)","process":{"pid":11842,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":23916544,"heapTotal":81956864,"heapUsed":75197616,"external":35935722}},"os":{"loadavg":[2.04296875,2.1455078125,2.3271484375],"uptime":269439},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Fri Sep 21 2018 17:01:47 GMT-0500 (EST)","process":{"pid":11842,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":23957504,"heapTotal":81956864,"heapUsed":75241448,"external":35935722}},"os":{"loadavg":[2.04296875,2.1455078125,2.3271484375],"uptime":269439},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Fri Sep 21 2018 17:01:48 GMT-0500 (EST)","process":{"pid":11842,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":24018944,"heapTotal":81956864,"heapUsed":75299832,"external":35943914}},"os":{"loadavg":[2.04296875,2.1455078125,2.3271484375],"uptime":269440},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Fri Sep 21 2018 17:01:48 GMT-0500 (EST)","process":{"pid":11842,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":24137728,"heapTotal":81956864,"heapUsed":75351912,"external":35943914}},"os":{"loadavg":[2.04296875,2.1455078125,2.3271484375],"uptime":269440},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Fri Sep 21 2018 17:01:51 GMT-0500 (EST)","process":{"pid":11842,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":24227840,"heapTotal":81956864,"heapUsed":75436432,"external":35944174}},"os":{"loadavg":[2.04296875,2.1455078125,2.3271484375],"uptime":269443},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":6379},"level":"error","message":"uncaughtException: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\nError: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","stack":"Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at Object._errnoException (util.js:999:13)\n    at _exceptionWithHostPort (util.js:1020:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1207:14)","exception":true,"date":"Fri Sep 21 2018 17:01:51 GMT-0500 (EST)","process":{"pid":11842,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":24272896,"heapTotal":81956864,"heapUsed":75480704,"external":35944174}},"os":{"loadavg":[2.04296875,2.1455078125,2.3271484375],"uptime":269443},"trace":[{"column":13,"file":"util.js","function":"Object._errnoException","line":999,"method":"_errnoException","native":false},{"column":20,"file":"util.js","function":"_exceptionWithHostPort","line":1020,"method":null,"native":false},{"column":14,"file":"net.js","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1207,"method":"afterConnect [as oncomplete]","native":false}]}
{"error":{},"level":"error","message":"uncaughtException: (0 , _chat2.default) is not a function\nTypeError: (0 , _chat2.default) is not a function\n    at init (/Users/adavia/Projects/node/statement/backend/src/channels/index.js:30:2)\n    at Object.<anonymous> (/Users/adavia/Projects/node/statement/backend/src/index.js:15:18)\n    at Module._compile (module.js:660:30)\n    at loader (/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js:144:5)\n    at Object.require.extensions.(anonymous function) [as .js] (/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js:154:7)\n    at Module.load (module.js:573:32)\n    at tryModuleLoad (module.js:513:12)\n    at Function.Module._load (module.js:505:3)\n    at Function.Module.runMain (module.js:701:10)\n    at Object.<anonymous> (/Users/adavia/Projects/node/statement/backend/node_modules/babel-cli/lib/_babel-node.js:154:22)\n    at Module._compile (module.js:660:30)\n    at Object.Module._extensions..js (module.js:671:10)\n    at Module.load (module.js:573:32)\n    at tryModuleLoad (module.js:513:12)\n    at Function.Module._load (module.js:505:3)\n    at Function.Module.runMain (module.js:701:10)","stack":"TypeError: (0 , _chat2.default) is not a function\n    at init (/Users/adavia/Projects/node/statement/backend/src/channels/index.js:30:2)\n    at Object.<anonymous> (/Users/adavia/Projects/node/statement/backend/src/index.js:15:18)\n    at Module._compile (module.js:660:30)\n    at loader (/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js:144:5)\n    at Object.require.extensions.(anonymous function) [as .js] (/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js:154:7)\n    at Module.load (module.js:573:32)\n    at tryModuleLoad (module.js:513:12)\n    at Function.Module._load (module.js:505:3)\n    at Function.Module.runMain (module.js:701:10)\n    at Object.<anonymous> (/Users/adavia/Projects/node/statement/backend/node_modules/babel-cli/lib/_babel-node.js:154:22)\n    at Module._compile (module.js:660:30)\n    at Object.Module._extensions..js (module.js:671:10)\n    at Module.load (module.js:573:32)\n    at tryModuleLoad (module.js:513:12)\n    at Function.Module._load (module.js:505:3)\n    at Function.Module.runMain (module.js:701:10)","exception":true,"date":"Fri Sep 21 2018 17:45:06 GMT-0500 (EST)","process":{"pid":12354,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":156037120,"heapTotal":136278016,"heapUsed":103390616,"external":36375961}},"os":{"loadavg":[2.49951171875,2.39892578125,2.25244140625],"uptime":272038},"trace":[{"column":2,"file":"/Users/adavia/Projects/node/statement/backend/src/channels/index.js","function":"init","line":30,"method":null,"native":false},{"column":18,"file":"/Users/adavia/Projects/node/statement/backend/src/index.js","function":null,"line":15,"method":null,"native":false},{"column":30,"file":"module.js","function":"Module._compile","line":660,"method":"_compile","native":false},{"column":5,"file":"/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js","function":"loader","line":144,"method":null,"native":false},{"column":7,"file":"/Users/adavia/Projects/node/statement/backend/node_modules/babel-register/lib/node.js","function":"Object.require.extensions.(anonymous function) [as .js]","line":154,"method":"js]","native":false},{"column":32,"file":"module.js","function":"Module.load","line":573,"method":"load","native":false},{"column":12,"file":"module.js","function":"tryModuleLoad","line":513,"method":null,"native":false},{"column":3,"file":"module.js","function":"Module._load","line":505,"method":"_load","native":false},{"column":10,"file":"module.js","function":"Module.runMain","line":701,"method":"runMain","native":false},{"column":22,"file":"/Users/adavia/Projects/node/statement/backend/node_modules/babel-cli/lib/_babel-node.js","function":null,"line":154,"method":null,"native":false},{"column":30,"file":"module.js","function":"Module._compile","line":660,"method":"_compile","native":false},{"column":10,"file":"module.js","function":"Module._extensions..js","line":671,"method":".js","native":false},{"column":32,"file":"module.js","function":"Module.load","line":573,"method":"load","native":false},{"column":12,"file":"module.js","function":"tryModuleLoad","line":513,"method":null,"native":false},{"column":3,"file":"module.js","function":"Module._load","line":505,"method":"_load","native":false},{"column":10,"file":"module.js","function":"Module.runMain","line":701,"method":"runMain","native":false}]}
{"error":{},"level":"error","message":"uncaughtException: categoryId is not defined\nReferenceError: categoryId is not defined\n    at Namespace.<anonymous> (/Users/adavia/Projects/node/statement/backend/src/channels/chat.channel.js:31:45)\n    at Namespace.emit (events.js:159:13)\n    at Namespace.emit (/Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js:212:10)\n    at /Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js:180:14\n    at process._tickCallback (internal/process/next_tick.js:150:11)","stack":"ReferenceError: categoryId is not defined\n    at Namespace.<anonymous> (/Users/adavia/Projects/node/statement/backend/src/channels/chat.channel.js:31:45)\n    at Namespace.emit (events.js:159:13)\n    at Namespace.emit (/Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js:212:10)\n    at /Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js:180:14\n    at process._tickCallback (internal/process/next_tick.js:150:11)","exception":true,"date":"Tue Sep 25 2018 17:53:12 GMT-0500 (EST)","process":{"pid":2610,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":150310912,"heapTotal":124174336,"heapUsed":85615560,"external":40600381}},"os":{"loadavg":[3.23681640625,5.9033203125,5.80419921875],"uptime":16186},"trace":[{"column":45,"file":"/Users/adavia/Projects/node/statement/backend/src/channels/chat.channel.js","function":null,"line":31,"method":null,"native":false},{"column":13,"file":"events.js","function":"Namespace.emit","line":159,"method":"emit","native":false},{"column":10,"file":"/Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js","function":"Namespace.emit","line":212,"method":"emit","native":false},{"column":14,"file":"/Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js","function":null,"line":180,"method":null,"native":false},{"column":11,"file":"internal/process/next_tick.js","function":"process._tickCallback","line":150,"method":"_tickCallback","native":false}]}
{"error":{},"level":"error","message":"uncaughtException: categoryId is not defined\nReferenceError: categoryId is not defined\n    at Namespace.<anonymous> (/Users/adavia/Projects/node/statement/backend/src/channels/chat.channel.js:31:24)\n    at Namespace.emit (events.js:159:13)\n    at Namespace.emit (/Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js:212:10)\n    at /Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js:180:14\n    at process._tickCallback (internal/process/next_tick.js:150:11)","stack":"ReferenceError: categoryId is not defined\n    at Namespace.<anonymous> (/Users/adavia/Projects/node/statement/backend/src/channels/chat.channel.js:31:24)\n    at Namespace.emit (events.js:159:13)\n    at Namespace.emit (/Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js:212:10)\n    at /Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js:180:14\n    at process._tickCallback (internal/process/next_tick.js:150:11)","exception":true,"date":"Tue Sep 25 2018 17:53:58 GMT-0500 (EST)","process":{"pid":2614,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":151179264,"heapTotal":121552896,"heapUsed":80485904,"external":40600717}},"os":{"loadavg":[3.40185546875,5.58642578125,5.69140625],"uptime":16232},"trace":[{"column":24,"file":"/Users/adavia/Projects/node/statement/backend/src/channels/chat.channel.js","function":null,"line":31,"method":null,"native":false},{"column":13,"file":"events.js","function":"Namespace.emit","line":159,"method":"emit","native":false},{"column":10,"file":"/Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js","function":"Namespace.emit","line":212,"method":"emit","native":false},{"column":14,"file":"/Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js","function":null,"line":180,"method":null,"native":false},{"column":11,"file":"internal/process/next_tick.js","function":"process._tickCallback","line":150,"method":"_tickCallback","native":false}]}
{"error":{},"level":"error","message":"uncaughtException: categoryId is not defined\nReferenceError: categoryId is not defined\n    at Namespace.<anonymous> (/Users/adavia/Projects/node/statement/backend/src/channels/chat.channel.js:31:24)\n    at Namespace.emit (events.js:159:13)\n    at Namespace.emit (/Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js:212:10)\n    at /Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js:180:14\n    at process._tickCallback (internal/process/next_tick.js:150:11)","stack":"ReferenceError: categoryId is not defined\n    at Namespace.<anonymous> (/Users/adavia/Projects/node/statement/backend/src/channels/chat.channel.js:31:24)\n    at Namespace.emit (events.js:159:13)\n    at Namespace.emit (/Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js:212:10)\n    at /Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js:180:14\n    at process._tickCallback (internal/process/next_tick.js:150:11)","exception":true,"date":"Tue Sep 25 2018 17:54:00 GMT-0500 (EST)","process":{"pid":2614,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":135733248,"heapTotal":121552896,"heapUsed":81593816,"external":40604568}},"os":{"loadavg":[3.40185546875,5.58642578125,5.69140625],"uptime":16234},"trace":[{"column":24,"file":"/Users/adavia/Projects/node/statement/backend/src/channels/chat.channel.js","function":null,"line":31,"method":null,"native":false},{"column":13,"file":"events.js","function":"Namespace.emit","line":159,"method":"emit","native":false},{"column":10,"file":"/Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js","function":"Namespace.emit","line":212,"method":"emit","native":false},{"column":14,"file":"/Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js","function":null,"line":180,"method":null,"native":false},{"column":11,"file":"internal/process/next_tick.js","function":"process._tickCallback","line":150,"method":"_tickCallback","native":false}]}
{"error":{},"level":"error","message":"uncaughtException: categoryId is not defined\nReferenceError: categoryId is not defined\n    at Namespace.<anonymous> (/Users/adavia/Projects/node/statement/backend/src/channels/chat.channel.js:31:24)\n    at Namespace.emit (events.js:159:13)\n    at Namespace.emit (/Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js:212:10)\n    at /Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js:180:14\n    at process._tickCallback (internal/process/next_tick.js:150:11)","stack":"ReferenceError: categoryId is not defined\n    at Namespace.<anonymous> (/Users/adavia/Projects/node/statement/backend/src/channels/chat.channel.js:31:24)\n    at Namespace.emit (events.js:159:13)\n    at Namespace.emit (/Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js:212:10)\n    at /Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js:180:14\n    at process._tickCallback (internal/process/next_tick.js:150:11)","exception":true,"date":"Tue Sep 25 2018 17:54:48 GMT-0500 (EST)","process":{"pid":2614,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":92119040,"heapTotal":76177408,"heapUsed":70121160,"external":35725362}},"os":{"loadavg":[2.529296875,5.01416015625,5.47265625],"uptime":16282},"trace":[{"column":24,"file":"/Users/adavia/Projects/node/statement/backend/src/channels/chat.channel.js","function":null,"line":31,"method":null,"native":false},{"column":13,"file":"events.js","function":"Namespace.emit","line":159,"method":"emit","native":false},{"column":10,"file":"/Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js","function":"Namespace.emit","line":212,"method":"emit","native":false},{"column":14,"file":"/Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js","function":null,"line":180,"method":null,"native":false},{"column":11,"file":"internal/process/next_tick.js","function":"process._tickCallback","line":150,"method":"_tickCallback","native":false}]}
{"error":{},"level":"error","message":"uncaughtException: Cannot read property 'user' of undefined\nTypeError: Cannot read property 'user' of undefined\n    at Namespace.<anonymous> (/Users/adavia/Projects/node/statement/backend/src/channels/chat.channel.js:10:37)\n    at Namespace.emit (events.js:159:13)\n    at Namespace.emit (/Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js:212:10)\n    at /Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js:180:14\n    at process._tickCallback (internal/process/next_tick.js:150:11)","stack":"TypeError: Cannot read property 'user' of undefined\n    at Namespace.<anonymous> (/Users/adavia/Projects/node/statement/backend/src/channels/chat.channel.js:10:37)\n    at Namespace.emit (events.js:159:13)\n    at Namespace.emit (/Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js:212:10)\n    at /Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js:180:14\n    at process._tickCallback (internal/process/next_tick.js:150:11)","exception":true,"date":"Wed Sep 26 2018 13:59:40 GMT-0500 (EST)","process":{"pid":4997,"uid":501,"gid":20,"cwd":"/Users/adavia/Projects/node/statement/backend","execPath":"/Users/adavia/.nvm/versions/node/v9.3.0/bin/node","version":"v9.3.0","argv":["node","/Users/adavia/Projects/node/statement/backend/src/index.js"],"memoryUsage":{"rss":31932416,"heapTotal":81420288,"heapUsed":77563728,"external":35838568}},"os":{"loadavg":[42.53955078125,23.5654296875,10.99951171875],"uptime":88569},"trace":[{"column":37,"file":"/Users/adavia/Projects/node/statement/backend/src/channels/chat.channel.js","function":null,"line":10,"method":null,"native":false},{"column":13,"file":"events.js","function":"Namespace.emit","line":159,"method":"emit","native":false},{"column":10,"file":"/Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js","function":"Namespace.emit","line":212,"method":"emit","native":false},{"column":14,"file":"/Users/adavia/Projects/node/statement/backend/node_modules/socket.io/lib/namespace.js","function":null,"line":180,"method":null,"native":false},{"column":11,"file":"internal/process/next_tick.js","function":"process._tickCallback","line":150,"method":"_tickCallback","native":false}]}
